.github/workflows/rust.yml

```diff
@@ -20,7 +20,7 @@ jobs:
   tests:
     strategy:
       matrix:
-        rust_version: [1.67.0, stable]
+        rust_version: [1.67, stable]
```

Comment:
error: package `time v0.3.37` cannot be built because it requires rustc 1.67.1 or newer, while the currently active rustc version is 1.67.0